src/backend/shared/styles/globals.css

```
@@ -258,6 +258,506 @@
+     raised : inset -1px -1px #0a0a0a, inset 1px 1px #fff,
+              inset -2px -2px #808080, inset 2px 2px #dfdfdf
+     pressed: inset -1px -1px #fff,    inset 1px 1px #0a0a0a,
+              inset -2px -2px #dfdfdf, inset 2px 2px #808080
+     sunken : inset -1px -1px #fff,    inset 1px 1px #808080,
+              inset -2px -2px #dfdfdf, inset 2px 2px #0a0a0a
+   ========================================================================== */
+
+/* Crisp, aliased period typeface across the whole app. */
+.nineties,
+.nineties * {
+  font-family: 'Tahoma', 'Geneva', 'Segoe UI', 'MS Sans Serif', system-ui, sans-serif !important;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Tahoma" (font-family-name-quotes)

(font-family-name-quotes)